pkg/ddc/alluxio/transform.go

```diff
@@ -299,6 +299,7 @@ func (e *AlluxioEngine) transformMasters(runtime *datav1alpha1.AlluxioRuntime,
 
 	if len(runtime.Spec.Master.Properties) > 0 {
 		value.Master.Properties = runtime.Spec.Master.Properties
+		e.overrideProperties(runtime, value.Master.Properties)
```

Comment:
I suggest to reuse utility function [`UnionMapsWithOverride`](https://github.com/fluid-cloudnative/fluid/blob/master/pkg/utils/map.go#L21) to fix this.

Reply:
done